bucket/prepare_data.py

```diff
@@ -64,7 +64,7 @@ def createDir(base, dirName):
     ############################################################################
     # Create TF Records
     ############################################################################
-    image_files = [image for image in annotations.keys()]
+    image_files = [image for image in annotations.keys() if image not in image_files_exceptions]
```

Comment:
The prepare script is used by watson machine learning as well, so this will only half fix the problem. We should instead just check if each of the image files listed in annotations.json are found in the `read_bucket` path